txpool/txpool.go

```diff
@@ -46,6 +46,7 @@ var (
 	ErrAlreadyKnown            = errors.New("already known")
 	ErrOversizedData           = errors.New("oversized data")
 	ErrMaxEnqueuedLimitReached = errors.New("maximum number of enqueued transactions reached")
+	ErrRejectFutureTx          = errors.New("rejected future tx due to low memory")
```

Comment:
Why memory instead of the term, slots? I don't think memory usage is related to this error

Reply:
Well, slots is the pool's concept of memory. 

I'll rename it to slots.
[Renamed](https://github.com/0xPolygon/polygon-edge/pull/690/commits/7f66b77089d1b2d378435a40a9100cd05888110a)